tests/functional/test_warning.py

```diff
@@ -2,7 +2,6 @@
 
 
 def test_environ(script, tmpdir):
-    """$PYTHONWARNINGS was added in python2.7"""
```

Comment:
I think this was meant to describe what the test is doing. Moving this below makes it seem like the test is meaningful on its own and PYTHONWARNINGS was added later. I would update this docstring to be e.g. "Test PYTHONWARNINGS, as added in python2.7" or remove the docstring and update the test name to `test_PYTHONWARNINGS`.

Reply:
Yea... However, given that 2.7 is the lowest version we have, it doesn't describe much. Anyway, IMO this is one of the doc-strings that don't add value by being there (it doesn't describe the test).

We should do follow up PR, adding a more useful docstring -- I'll merge, since `test_warnings.py::test_environ` probably explains enough about this test, for now?